src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected String by;
+    protected LocalDate localBy;
```

Comment:
Perhaps the name of this LocalDate variable could also be replaced by a clearer one?